### PR TITLE
If else to Switch case and add "info" command

### PR DESCRIPTION
**Pull request by duongoku**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,9 +10,27 @@ client.on('message', async (msg) => {
   const command = args.shift().slice(prefix.length);
   //Add your commands here. Good command handlers are overrated :POGGERS:
   
-  if(command === 'ping'){
-    const m = await msg.channel.send('Pinging...');
-    m.edit(`Ponggers! Client ping: \`${m.createdTimestamp - msg.createdTimestamp}\`ms. Heartbeat ping: \`${client.ping}\`ms`);
+  switch(command){
```
Comment (Keswiik) [inline index.js]: At least use decent formatting.

File: index.js [hunk 2]
```diff
@@ -10,9 +10,27 @@ client.on('message', async (msg) => {
...
+
+  	case "ping":
+
+    	const m = await msg.channel.send('Pinging...');
+    	m.edit(`Ponggers! Client ping: \`${m.createdTimestamp - msg.createdTimestamp}\`ms. Heartbeat ping: \`${client.ping}\`ms`);\
+    	
+    	break;
+
+    case "info":
+    	var embed = new Discord.RichEmbed()
```
Comment (Keswiik) [inline index.js]: `const` 🤔

**Review by Keswiik — CHANGES_REQUESTED** [review]

**Comment by lunamana104** [comment]
ok, I'll fix them

**Comment by duongoku** [comment]
> ok, I'll fix them

thanks dude

**Comment by RuyiLi** [comment]
`Discord` is undefined

**Comment by duongoku** [comment]
Oops, I'll fix this again xD

**Comment by duongoku** [comment]
> `Discord` is undefined

Can you review my code again? I've also fix lunamana104 code

**Pull request closed without merging**